### ISSUE TITLE
508-defect-1 ❗ Launchblocker [SCREENREADER, COGNITION]: Collectively major structure, heading, and organization issues on the main page "Spotlight"

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#25019]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

## User Story or Problem Statement:
As a screen reader user, I expect web pages to be organized logically. I expect to determine based on HTML elements how content is related, and I expect to determine a link's purpose and target by its link text.

<details>
<summary> Adding screenshot for reference </summary>

![image](https://user-images.githubusercontent.com/55411834/119573125-110dc000-bd71-11eb-8cda-80ff0bb805b3.png)

</details>

## Details
This page has several issues with structure, organization, etc. which cumulatively make it impossible for a screen reader user to interpret content.

- Items under "in the spotlight" are in a list element whose bullets are not read by JAWS with no distinguishing mark-up, making it difficult to determine if item three is a part of or separate from item two.
- Under "In the spotlight," in the list of new groups, JAWS reads a series of words and short phrases. It is impossible to determine which are descriptors of a group and which are names of different groups.
- Under "In the spotlight," in the list of new groups, only one group is linked to its corresponding web page.
- Under "In the Spotlight," the link text "Listen to the podcast" doesn't specify which podcast the user is listening too, increasing the cognitive load on those who navigate by links

## Acceptance Criteria _(updated by Michelle)_
- [x] Titles of items under "In the spotlight" are tagged with level 3 headings.

**^ After addressing the heading level, we will follow up with Angela for next steps.** 

---

~~Links under "In the spotlight" indicate purpose and target.~~ _(likely will be addressed from content side but will get heading level change in place first and then follow up with Angela)_
~~List of new groups is structured such that groups can be distinguished from each other.~~_(likely will be addressed from content side but will get heading level change in place first and then follow up with Angela)_
~~All group names are linked to their corresponding web pages.~~ _(removing this AC because linking is not a requirement for content editors and was done in this case for testing FE implementation)_

**Comment by mmiddaugh** [comment]
Thank you for this feedback, @AngelaFowler82 
I separated the feedback for the "Locations" section into a separate issue, which is #25172, for your reference

**Comment by mmiddaugh** [comment]
@AngelaFowler82 @joshkimux please confirm if my understanding of the expectations is complete and correct - 
- "In the spotlight at Escanaba Vet Center" should be an H2
- the title in each individual spotlight (such as "Borne the Battle Podcast: Vet Centers" should be H3)
- We're assuming the use of <li> in cases when there is no CTA is causing some of the contextual confusion - will using a <div> and rich text help with the organizational issues? 
- We'll need to address issues beyond this with content writing recommendations for staff editors

**Comment by AngelaFowler82** [comment]
Headings will make the section far more understandable. Once those are in place I would like to take another look at the cards, particularly the one concerning groups. Alternatively, maybe it would be helpful is Michelle and I wend over it during our meeting today?

**Comment by AngelaFowler82** [comment]
Has this been migrated to staging yet? Socks is not my friend. *sigh*

**Comment by mpelzsherman** [comment]
Can someone please add some links to this ticket so we are clear about which page(s) we're talking about? Thanks.

**Comment by mpelzsherman** [comment]
Never mind about the links -  this is on the main Vet Center page. I've merged a PR. We will do another tugboat build soon.

**Comment by mmiddaugh** [comment]
Will have @AngelaFowler82 validate this issue once we have a Vet Center published and available on Staging.

**Comment by mmiddaugh** [comment]
This can now be validated in Prod 🎉 
https://www.va.gov/nashville-vet-center/
https://www.va.gov/colorado-springs-vet-center/
@AngelaFowler82 @joshkimux

**Comment by AngelaFowler82** [comment]
Headings are properly nested and each article is clearly marked. Are the podcast and specific groups no longer highlighted on the vet center page about which this issue was created? If so, we can close.

**Comment by mmiddaugh** [comment]
@AngelaFowler82 
The podcast spotlight has been replaced with new national content titled "Vet Centers are celebrating their 42nd anniversary!" The podcast content from the staging review is similar to Colorado Springs' content  titled "Military Arts Connection (MAC) Program Partnership" in that both have calls to action and link to a page outside VA.gov. 

We do have a few Vet Center pages which include a group list in one of the spotlights. Once one of those is published and available in Prod, I think we should have you confirm that it is announced appropriately by screen reader. 

The spotlights will be used in a variety of ways and I think it is important to make sure screen readers handle each type of content clearly.
